modules/samples/sample-clients/apim-event-client/src/main/java/org/wso2/carbon/sample/apimevent/DataPublisherUtil.java

```diff
@@ -0,0 +1,81 @@
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.sample.apimevent;
+
+import java.io.File;
+
+/**
+ * Datapublisher Util Implementataion.
+ */
+public class DataPublisherUtil {
+
+    public static void setTrustStoreParams() {
+        File filePath = new File("src" + File.separator + "main" + File.separator + "resources");
+        if (!filePath.exists()) {
+            filePath = new File("modules" + File.separator + "samples" + File.separator +
+                    "sample-clients" + File.separator + "apimevent-client" + File.separator + "src" +
+                    File.separator + "main" + File.separator + "resources");
+        }
+        if (!filePath.exists()) {
+            filePath = new File("resources");
+        }
+        if (!filePath.exists()) {
+            filePath = new File("test" + File.separator + "resources");
+        }
+        String trustStore = filePath.getAbsolutePath();
+        System.setProperty("javax.net.ssl.trustStore", trustStore + File.separator + "client-truststore.jks");
+        System.setProperty("javax.net.ssl.trustStorePassword", "wso2carbon");
+
+    }
+
+    public static void setKeyStoreParams() {
+        File filePath = new File("src" + File.separator + "main" + File.separator + "resources");
+        if (!filePath.exists()) {
+            filePath = new File("modules" + File.separator + "samples" + File.separator +
+                    "sample-clients" + File.separator + "apimevent-client" + File.separator + "src" +
+                    File.separator + "main" + File.separator + "resources");
+        }
+        if (!filePath.exists()) {
+            filePath = new File("resources");
+        }
+        if (!filePath.exists()) {
+            filePath = new File("test" + File.separator + "resources");
+        }
+        String keyStore = filePath.getAbsolutePath();
+        System.setProperty("Security.KeyStore.Location", keyStore + File.separator + "wso2carbon.jks");
+        System.setProperty("Security.KeyStore.Password", "wso2carbon");
+    }
+
+    public static String getDataAgentConfigPath(String fileName) {
+        File filePath = new File("src" + File.separator + "main" + File.separator + "resources");
+        if (!filePath.exists()) {
+            filePath = new File("modules" + File.separator + "samples" + File.separator +
+                    "sample-clients" + File.separator + "apimevent-client" + File.separator + "src" +
+                    File.separator + "main" + File.separator + "resources");
+        }
+        if (!filePath.exists()) {
+            filePath = new File("resources");
+        }
+        if (!filePath.exists()) {
+            filePath = new File("test" + File.separator + "resources");
+        }
+        return filePath.getAbsolutePath() + File.separator + fileName;
```

Comment:
There are some string used multiple times. Can we make them class level constants?

Reply:
Fixed in 1f69fe0038ac7ea895a7fae7e7714add8fc82281